lib/assert.js

```diff
@@ -131,8 +129,8 @@ function areSimilarTypedArrays(a, b) {
     }
     return true;
   }
-  return compare(from(a.buffer, a.byteOffset, len),
```

Comment:
I am no sure I can follow. The Buffer.from part was removed intentionally. Should I add a comment to describe the Uint8Array part?